<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="button_job">
     <property name="text">
      <string>Xuất file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="button_confirm">
     <property name="text">
      <string>Xác nhận thông tin</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_field">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Times New Roman&quot;;
</string>
         </property>
         <property name="text">
          <string>Chọn nghành:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_field"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_field">
         <property name="text">
          <string>nhấn khi chọn xong</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_job">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Chọn nghề:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_job"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_job_3">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Số năm kinh nghiệm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_compare">
        <item>
         <property name="text">
          <string>Bằng</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trên</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dưới</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_yearexp">
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Information">
     <property name="text">
      <string>Mời bạn chọn !</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
